src/NuGet.Clients/Options/GeneralOptionControl.cs

```diff
@@ -98,5 +101,26 @@ private void packageRestoreConsentCheckBox_CheckedChanged(object sender, EventAr
                 packageRestoreAutomaticCheckBox.Checked = false;
             }
         }
+
+        private void localsCommandButton_OnClick(object sender, EventArgs e)
+        {
+            var arguments = new List<string>();
+            arguments.Add("all");
+            var settings = ServiceLocator.GetInstance<ISettings>();
+            var localsCommandRunner = new LocalsCommandRunner(arguments , settings, clear:true, list:false);
+            try
+            {
+                localsCommandRunner.ExecuteCommand();
+            }
+            catch(ArgumentException ex)
+            {
+                MessageBox.Show(ex.Message, Resources.PopupTitle_ClearNuGetCache);
+            }
+            if (localsCommandRunner.Result == LocalsCommandRunner.LocalsCommandResult.ClearSuccess)
```

Comment:
A message box/popup. Same as below, but with the exception message (Which will be the same as the message displayed for cli errors)

Ref - 

![2016-08-17](https://cloud.githubusercontent.com/assets/10507120/17746904/34d983f6-6467-11e6-8092-682dae58eea9.png)